.env.docker.dev

```diff
@@ -0,0 +1,33 @@
+# Copy this file to ".env.dev" and ".env.docker.dev", and then tweak values for local development
```

Comment:
Question: Do we need this file in the repo? I understood that it was to be copied from env.template.

Reply:
Following the pattern of the other repos, most have it committed. I somewhat agree, but I think we need further review & agreement on how this should work. Maybe we should provide multiple `.env.*.template` files, then, for different scenarios. Let's consider it out of scope to change the pattern in this ticket.